joplin/base/context_processors.py

```diff
@@ -2,4 +2,7 @@
 
 
 def settings_context(request):
```

Comment:
The context processor is Django's preferred way of passing constants into the app for templates